### PR TITLE
Feature/table graph sort from graph

### PR DESCRIPTION
**Pull request by ebb-tide**
- [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #

### The problem
Adds click to sort behavior to the table field names.

### CODE REVIEW

File: ui/src/shared/components/TableGraph.js [hunk 1]
```diff
@@ -80,25 +110,55 @@ class TableGraph extends Component {
     }
   }
 
+  handleClickFieldName = (columnIndex, rowIndex) => () => {
+    const {tableOptions} = this.props
+    const {clicktoSortFieldIndex, clicktoSortDirection, data} = this.state
```
Comment (ischolten) [inline ui/src/shared/components/TableGraph.js]: Variable not in camelCase, shouldnt it be 'clickToSort...'?

File: ui/src/shared/components/TableGraph.js [hunk 2]
```diff
@@ -25,21 +27,49 @@ class TableGraph extends Component {
...
+      sortByColumnIndex: NULL_COLUMN_INDEX,
+      clicktoSortFieldIndex: NULL_COLUMN_INDEX,
+      clicktoSortDirection: DEFAULT_SORT,
     }
   }
 
   componentWillReceiveProps(nextProps) {
-    const {data, unzippedData} = timeSeriesToTableGraph(nextProps.data)
-
+    const {data} = timeSeriesToTableGraph(nextProps.data)
```
Comment (ischolten) [inline ui/src/shared/components/TableGraph.js]: are you getting unzippedData from timeSeriesToTableGraph anymore? you can probably stop it from doing that calculation there so as not to duplicate work

File: ui/src/style/components/table-graph.scss [hunk 3]
```diff
@@ -67,6 +67,18 @@
...
   }
+  &__isFieldName:hover {
+    cursor: pointer;
+  }
+  &__isFieldName:before {
+    font-family: 'icomoon';
+    content: "\e902";
+    font-size: 17px;
+  }
+  &__isSortedUp {
```
Comment (ischolten) [inline ui/src/style/components/table-graph.scss]: eventually these need something in them.

File: ui/src/shared/constants/tableGraph.js [hunk 4]
```diff
@@ -8,6 +8,9 @@ export const TIME_FORMAT_CUSTOM = 'Custom'
 
 export const TIME_COLUMN_DEFAULT = {internalName: 'time', displayName: ''}
 
+export const DEFAULT_SORT = 'asc'
+export const ALT_SORT = 'desc'
```
Comment (ischolten) [inline ui/src/shared/constants/tableGraph.js]: these constants could be `ASCENDING` and `DESCENDING` and then DEFAULT_SORT could be one or the other

File: ui/src/shared/components/TableGraph.js (hunk 1 above)
Comment (ebb-tide) [reply ui/src/shared/components/TableGraph.js]: thank you.

File: ui/src/style/components/table-graph.scss (hunk 3 above)
Comment (ebb-tide) [reply ui/src/style/components/table-graph.scss]: yes they do.

File: ui/src/shared/constants/tableGraph.js (hunk 4 above)
Comment (ebb-tide) [reply ui/src/shared/constants/tableGraph.js]: ok. 'asc' and 'desc' is what lodash requires. but yes it's nice to make it readable

File: ui/src/shared/components/TableGraph.js (hunk 2 above)
Comment (ebb-tide) [reply ui/src/shared/components/TableGraph.js]: good point thanks.

File: ui/src/style/components/table-graph.scss (hunk 3 above)
Comment (alexpaxton) [reply ui/src/style/components/table-graph.scss]: What uppppp

Comment (ebb-tide) [reply ui/src/style/components/table-graph.scss]: hahah.. yes allex?? we can do this whenever you are not hanging out with Iris

**Pull request merged**